src/initial_conditions/initial_conditions.jl

```diff
@@ -16,6 +16,105 @@ perturb_coeff(p::Geometry.LatLongZPoint{FT}) where {FT} = sind(p.long)
 perturb_coeff(p::Geometry.XZPoint{FT}) where {FT} = sin(p.x)
 perturb_coeff(p::Geometry.XYZPoint{FT}) where {FT} = sin(p.x)
 
+struct ColumnInterpolatableField{F, D}
+    f::F
+    data::D
+    function ColumnInterpolatableField(f::Fields.ColumnField)
+        zdata = vec(parent(Fields.Fields.coordinate_field(f).z))
+        fdata = vec(parent(f))
+        @assert length(zdata) == length(fdata)
```

Comment:
Just curious, will this ever be false if zdata is derived from coordinate_field(f).z?

Reply:
Hm, no I don't think so. I think the interface was previously different and it was possible. This should always be true, so I'll remove the assertion, thanks!